Sun Mar 10 00:19:53 FET 2013  Sergei Trofimovich <slyfox@community.haskell.org>
  * ghc-7.6: add missing Num instance for Bits
  
  Fixes the following build failure:
      src/runtime/haskell/Data/Binary/IEEE754.lhs:256:17:
          Could not deduce (Num a) arising from a use of `mask'
          from the context (Bits a)
            bound by the type signature for
                       clamp :: Bits a => BitCount -> a -> a
  
diff -rN -u old-GF/src/runtime/haskell/Data/Binary/IEEE754.lhs new-GF/src/runtime/haskell/Data/Binary/IEEE754.lhs
--- old-GF/src/runtime/haskell/Data/Binary/IEEE754.lhs	2013-03-10 00:42:54.232642899 +0300
+++ new-GF/src/runtime/haskell/Data/Binary/IEEE754.lhs	2013-03-10 00:42:56.279643003 +0300
@@ -252,7 +252,7 @@
 in positions above the count.
 
 \begin{code}
-clamp :: Bits a => BitCount -> a -> a
+clamp :: (Num a, Bits a) => BitCount -> a -> a
 clamp = (.&.) . mask where
 	mask 1 = 1
 	mask n | n > 1 = (mask (n - 1) `shiftL` 1) + 1
